### DISCUSSION TITLE
DHCP in Auto-configuration section

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#34] [General]
Hi Brian,
I believe that people have the right to know 2 utterly important facts:
- the most popular OS in the world (by the number of installations) does not support DHCPv6.
- DHCPv6 is filtered out by all 3GPP modems.
Hence, SLAAC is much more important than just "recommended".
Eduard

**Comment by EduardVasilenko** [comment]
sorry, reading sections in the order.
Found the Android discussion in the next session (Managed configuration).
But DHCP filtered in the Mobile environment still missing.

**Comment by becarpenter** [comment]
Is that described somewhere? In 3GPP for example?

**Comment by EduardVasilenko** [comment]
Good point. I have requested the most vocal people about the problem. You are on copy.
See yourself: would they present enough evidence.
Up to now they have shown:
- case "03085601" with Qualcomm
- draft-shytyi-v6ops-danir-05 section 5.5
- personal tests
Ed/

**Comment by becarpenter** [comment]
Will fix based on email discussion, thanks

**Comment by becarpenter** [comment]
Closed as done
